modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/loader/validation/ParameterModelValidatorTestCase.java

```diff
@@ -104,12 +103,4 @@ public void invalidModelDueToDefaultValueWhenRequired() {
     validate(extensionModel, validator);
   }
 
-  @Test(expected = IllegalModelDefinitionException.class)
-  public void invalidModelDueToListWithoutPluralName() {
```

Comment:
this is wrong. You only made the plural rule specific to the java loader. That doesn't mean you shouldn't have test for it. 

Also, where's the SourceDeclarationEnricher you removed in the ext-api PR?